_linter/markdown-linter-style.rb

```diff
@@ -0,0 +1,33 @@
+#  Include all rules by default
+all
+
+#  Rule Modifications
+#  --------------------------------
+## MD029 mod - tells linter not to fail if order list items don't start with 1
+rule 'MD029', :style => :ordered
+
+#  Exclude these rules
+#  --------------------------------
+## MD001 - Check that Header levels increment by one level at a time.
```

Comment:
Can we reorganize this so that there's a list of rules we are excluding by design and then another list of rules we're excluding *for now* because our current content does not adhere to the rule? It looks like everything except for "line length" and "inline HTML" are rules we actually want to obey.